packages/compile-solidity-tests/test/fixture/default-box/truffle-config.js

```diff
@@ -9,9 +9,10 @@ module.exports = {
   },
   compilers: {
     solc: {
+      version: "0.6.9",
```

Comment:
Does this have implications for other tests?

Reply:
Probably best to update to latest whenever we can, also

Reply:
Good catch y'all. This actually doesn't need to be there at all. I thought Harry said we needed a compiler version below 0.8.6 to see this issue but reading his notes, we just need a Solidity version below that. For reference though, only the profiler test is using this fixture so solc version wouldn't have mattered. Removing!